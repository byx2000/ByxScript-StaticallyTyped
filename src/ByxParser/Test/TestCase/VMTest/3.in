int g1 = 243398;
double g2 = 23874.324;
int g3;
double g4;
int g5;
double g6;

function int f1(int n)
{
	return n;
}

function double f2(double x)
{
	return x;
}

function double f3()
{
	return 23.75;
}

function int f4()
{
	return 7744;
}

function void main()
{
	int i = f1(3.14);
	double j = f2(100);
	g1 = i;
	g2 = j;
	
	g3 = f3();
	g4 = f3();
	
	g5 = f4();
	g6 = f4();
}