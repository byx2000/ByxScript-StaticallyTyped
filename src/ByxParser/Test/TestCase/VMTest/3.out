i 3
d 100
i 23
d 23.75
i 7744
d 7744